compiler/rustc_trait_selection/src/traits/coherence.rs

```diff
@@ -998,6 +998,7 @@ impl<'a, 'tcx> ProofTreeVisitor<'tcx> for AmbiguityCausesVisitor<'a> {
 
         let Goal { param_env, predicate } = goal.goal();
 
+        // FIXME: no `replace_bound_vars_with_placeholders` fn exist
```

Comment:
This is https://doc.rust-lang.org/nightly/nightly-rustc/rustc_infer/infer/struct.InferCtxt.html#method.instantiate_binder_with_placeholders